src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -235,10 +189,6 @@ const DropDownMenu = React.createClass({
    */
   getInputNode() {
```

Comment:
Ohhhh, I see what you mean now. `DropdownMenu` -> `SelectField` -> `TextField`.  Geez :fearful: 

>  It bit me so hard I spent a day in hospital :laughing:

Thank you for saving me.